image/entrypoint.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# More safety, by turning some bugs into errors.
+# Without `errexit` you don’t need ! and can replace
+# PIPESTATUS with a simple $?, but I don’t do that.
+set -o errexit -o pipefail -o noclobber -o nounset
+
+echo "Entrypoint script for translator"
```

Comment:
translator -> generic-translator as it is module's name

Reply:
updated